commit 1bf209dd7e5a0f0001b3d1e3798093772bbd3fd3
Author:     Karolin Seeger <kseeger@samba.org>
AuthorDate: Thu Oct 4 11:43:20 2012 +0200
Commit:     Karolin Seeger <kseeger@samba.org>
CommitDate: Thu Oct 4 13:48:00 2012 +0200

    html docs: Remove link to Using Samba.
    
    Thanks to Christian Perrier <bubulle@debian.org> for reporting!
    
    Fix bug #7826 - HTML docs index file still points to Using Samba.
    
    Karolin
    
    Autobuild-User(master): Karolin Seeger <kseeger@samba.org>
    Autobuild-Date(master): Thu Oct  4 13:48:00 CEST 2012 on sn-devel-104
---
 docs-xml/htmldocs.html | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/docs-xml/htmldocs.html b/docs-xml/htmldocs.html
index 44fcc0f..6fb9e73 100644
--- a/docs-xml/htmldocs.html
+++ b/docs-xml/htmldocs.html
@@ -23,10 +23,6 @@
  <td valign="top">This book provides example configurations, it documents key aspects of Microsoft Windows networking, provides in-depth insight into the important configuration of Samba-3, and helps to put all of these into a useful framework.</td>
 </tr>
 <tr>
- <td valign="top"><a href="using_samba/toc.html">Using Samba</a>, 2nd Edition</td>
- <td valign="top"><i>Using Samba</i>, Second Edition is a comprehensive guide to Samba administration. It covers all versions of Samba from 2.0 to 2.2, including selected features from an alpha version of 3.0, as well as the SWAT graphical configuration tool. Updated for Windows 2000, ME, and XP, the book also explores Samba's new role as a primary domain controller and domain member server, its support for the use of Windows NT/2000/XP authentication and filesystem security on the host Unix system, and accessing shared files and printers from Unix clients.</td>
-</tr>
-<tr>
  <td valign="top"><a href="manpages/index.html">Man pages</a></td>
  <td valign="top">The Samba man pages in HTML.</td>
 </tr>
